eppo_metrics_sync/dbt_model_parser.py

```diff
@@ -0,0 +1,128 @@
+from collections import Counter
+from itertools import chain
+
+
+class DbtModelParser():
+
+    def __init__(self, model, dbt_model_prefix):
+        self.model = model
+        self.eppo_entities = []
+        self.eppo_timestamp = []
+        self.eppo_facts = []
+        self.eppo_properties = []
+        self.dbt_model_prefix = dbt_model_prefix
+    
+    def parse_columns(self):
+
+        for column in self.model.get('columns'):
+
+            tags = column.get('tags')
+
+            if tags:
+                column_entities = [t for t in tags if 'eppo_entity:' in t]
+
+                if column_entities:
+                    for column_entity in column_entities:
+                        column_entity_split = column_entity.split(':')
+                        if len(column_entity_split) != 2:
+                            raise ValueError(
```

Comment:
Instead of this if/else logic, consider using `assert`.  So instead of 

```
if len(column_entity_split) != 2:
    raise ValueError(
        f'Invalid entity tag {column_entity} in model {self.model["name"]}'
    )
else:
    self.eppo_entities.append({
        "entity_name": column_entity_split[1],
        "column": column['name']
    })
```
Consider this:
```
EXPECTED_COLUMN_LENGTH = 2
assert len(column_entity_split) == EXPECTED_COLUMN_LENGTH, \
    f'Invalid entity tag {column_entity} in model {self.model["name"]}'

self.eppo_entities.append({
    "entity_name": column_entity_split[1],
    "column": column['name']
})
```
The second avoids the if/else, which makes it easier to read IMO. "Confirm this and move on" is a more direct way of saying "If there's an issue, stop; otherwise, move on"